app/src/main/java/io/intrepid/contest/screens/sendinvitations/selectcontacts/SelectContactsActivity.java

```diff
@@ -216,4 +218,23 @@ public void updateContactSearchFilter(String newFilter) {
         contactSearchString = newFilter;
         getSupportLoaderManager().restartLoader(CONTACTS_LOADER, null, this);
     }
+
+    @Override
+    public void notifyAdapterDataSetChanged() {
+        selectContactsAdapter.notifyDataSetChanged();
```

Comment:
I'm not a big fan of calling `notifyDataSetChanged()` by the adapter instance, but if this is how you guys have been doing it, it's fine.